lib/src/gestures/flutter_map_interactive_viewer.dart

```diff
@@ -366,6 +414,14 @@ class FlutterMapInteractiveViewerState
     }
   }
 
+  // Thanks to https://stackoverflow.com/questions/48916517/javascript-click-and-drag-to-rotate
+  double getCursorRotationDegrees(Offset offset, BuildContext context) =>
+      round((math.atan2(offset.dx - MediaQuery.sizeOf(context).width / 2,
+                  offset.dy - MediaQuery.sizeOf(context).height / 2) *
+              (180 / pi) *
+              -1) +
+          100);
```

Comment:
Maybe this would be more readable:
```dart
double getCursorRotationDegrees(Offset offset, BuildContext context) {
  final size = MediaQuery.sizeOf(context);
  return round((math.atan2(offset.dx - size.width / 2, offset.dy - size.height / 2) * (180 / math.pi) * -1) + 100);
}
```
Also I cannot seem to find `round(/* ... */)` method, usually you use the [`.round()`](https://api.dart.dev/stable/dart-core/double/round.html) and `(180 / pi)` shouldn't work as the import to `dart:math` is made with `as math` does this code compile for you? 🤔

Reply:
Here's a revised function:

```dart
// Thanks to https://stackoverflow.com/questions/48916517/javascript-click-and-drag-to-rotate
double getCursorRotationDegrees(Offset offset) {
  final size = MediaQuery.sizeOf(context);
  return -math.atan2(
          offset.dx - size.width / 2, offset.dy - size.height / 2) *
      (180 / math.pi);
}
```

It looks like `round` and `pi` came from 'latlong2'. I've switched to using `math.pi`.

Asking for the `context` as an argument turned out to be unnecessary.

I've also removed the rounding function, as I can't see a visible effect on the result. @ibrierley Can you confirm if you included the rounding for a reason?

I've also removed the `+ 100`, as that seems to be a correction term of sorts.
 - +180 or -180: map North when cursor top
 - +100: map North when cursor right
 - -100: map North when cursor left
 - 0: map North when cursor bottom

Not sure why the values are quite like that, or which has the best feeling? I guess +/- 180 kind of feels OK, as the cursor aligns with the North of the map. Open to your opinions!

Reply:
See 0a52a93 for resolution. I've added a local scope `correctionTerm` constant for easy identification, and set North = cursor (180).